src/lexer/error_handler.py

```diff
@@ -155,6 +163,13 @@ def __str__(self):
             log += f"{delim} "
         log += f"\n\tafter {self.temp_id} but got {self.actual_delim if self.actual_delim != ' ' else 'WHITESPACE'} instead\n"
 
+        # Error preview
+        index_str = str(self.position[0])
+        log += f"\t{'_' * 20}\n"
+        log += f"\t{index_str} | {ErrorSrc.src[self.position[0]]}\n"
+        log += f"\t{' ' * len(index_str)} | {' ' * self.position[1]}{'^'}\n"
+        log += f"\t{'_' * 20}\n"
```

Comment:
### [non-blocking]
for the top and bottom borders, thoughts on
```
log += f"\t{'_' * len(ErrorSrc.src[self.position[0]])}\n"
```
for longer borders depending on length of line. although this might mess with console since console wraps text around if line is too long. ALTHOUGH although, if the line is too long, it'll wrap around anyway

Reply:
That's true, would mess up the pointer either way hmmmmmmm

Reply:
#25